packages/discovery/src/discovery/config/ConfigReader.test.ts

```diff
@@ -2,8 +2,17 @@ import { expect } from 'earl'
 import { readdirSync } from 'fs'
 
 import { ConfigReader } from './ConfigReader'
+import path from 'path'
 
 describe(ConfigReader.name, () => {
+  // FIXME: This is a temporary hack to make the tests while configs are in backend
```

Comment:
This is here to make this PR work with minimal code changes. I expect the location of config.jsonc files to change in the future.